### PR TITLE
modify 3.x ipex example structure

### PR DESCRIPTION
**Pull request by violetch24**
## Type of Change
3.x example

## Description
modify 3.x ipex example structure, sq&static

## How has this PR been tested?
local test

### CODE REVIEW

**Comment by chensuyue** [comment]
Please add the new model path into: examples/.config/model_params_pytorch_3x.json

File: examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/smooth_quant/utils.py [hunk 1]
```diff
@@ -0,0 +1,193 @@
+import random
+import torch
+from collections import UserDict
+from packaging.version import Version
+from neural_compressor.common import logger
+from neural_compressor.torch.utils import get_torch_version
+
+class DataloaderPreprocessor:
```
Comment (xin3he) [inline examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/smooth_quant/utils.py:8]: Is this for gptq only? we can remove it in `smooth_quant`.

File: examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/static_quant/utils.py [hunk 2]
```diff
@@ -0,0 +1,193 @@
+import random
+import torch
+from collections import UserDict
+from packaging.version import Version
+from neural_compressor.common import logger
+from neural_compressor.torch.utils import get_torch_version
+
+class DataloaderPreprocessor:
```
Comment (xin3he) [inline examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/static_quant/utils.py:8]: the same as above

**Review by xin3he — APPROVED** [review]

File: docs/3x/PT_StaticQuant.md [hunk 3]
```diff
@@ -68,7 +68,7 @@ q_model = convert(prepared_model)
 
 #### Model Examples
 
-Users could refer to [examples](https://github.com/intel/neural-compressor/blob/master/examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/llm) on how to quantize a new model.
+Users could refer to [examples](https://github.com/intel/neural-compressor/blob/master/examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/static_quant) on how to quantize a new model.
```
Comment (Kaihui-intel) [inline docs/3x/PT_StaticQuant.md:71]: We should create an ipex folder under `static_quant` to distinguish between pt2e and ipex

**Review by Kaihui-intel — APPROVED** [review]

**Pull request merged**